test/smoke/process-usage/src/test/fixtures.js

```diff
@@ -0,0 +1,229 @@
+'use strict';
+
+const timewindow = require('abacus-timewindow');
+
+const { times } = require('underscore');
+
+const { fixturesCfg } = require('./env-config');
+
+// The scaling factor of each time window for creating the date string
+// [Second, Minute, Hour, Day, Month]
+const _slack = () =>
+  /^[0-9]+[MDhms]$/.test(fixturesCfg.slack)
+    ? {
+      scale: fixturesCfg.slack.charAt(fixturesCfg.slack.length - 1),
+      width: fixturesCfg.slack.match(/[0-9]+/)[0]
+    }
+    : {
+      scale: timewindow.dimension.min,
+      width: 10
+    };  
+
+const _initWindows = (win, dimension) => {
+  const windows = [win];
+
+  if(fixturesCfg.windowsSizes && fixturesCfg.windowsSizes[dimension])
+    times(fixturesCfg.windowsSizes[dimension] - 1, () => windows.push(null));
+
+  else {
+    const timeWindows = timewindow.timeWindowsSizes(_slack(), fixturesCfg.windowsSizes);
+    times(timeWindows.getWindows(dimension).length - 1, () => windows.push(null));
+  }
+
+  return windows;
+};
+
+const _buildExpectedWindows = (summary, quantity) => {
+
+  const addProperty = (key, value, obj) => {
+    if (value)
```

Comment:
Is this check necessary? Is `_buildExpectedWindows` ever called with a null/undefined summary or quantity?

Reply:
This function addProperty is not needed.